src/app/issuer/components/edit-qr-form/edit-qr-form.component.html

```diff
@@ -35,9 +35,9 @@ <h2 class="tw-hidden md:tw-block tw-text-purple tw-text-3xl">QR-Code Vergabe ers
                 <div *ngIf="!editing; else editButtons"
                     class="tw-flex tw-flex-col xl:tw-flex-row tw-gap-2 tw-justify-between xl:tw-items-end">
                     <div class="tw-flex md:tw-justify-between tw-gap-12">
-                        <oeb-input label="Start-Datum" [errorMessage]="'Bitte gebe ein Datum in der Form YYYY/mm/dd an'"
+                        <oeb-input labelStyle="tw-text-oebblack" label="Start-Datum" [errorMessage]="'Bitte gebe ein Datum in der Form YYYY/mm/dd an'"
```

Comment:
Grammatically it should be "Bitte gib ein Datum...". Also, it would be nice to have a translation here as well

Reply:
I fixed the grammar, we will do translations for the whole qr code feature later